style.css

```diff
@@ -0,0 +1,253 @@
+  border: none;
+  background-color: lightgray;
+  margin-right: 0;
+  margin-left: -10px;
+  padding-right: 40px;
+  padding-left: 10px;
+  width: 77%;
+}
+
+.search-button {
+  color: white;
+  font-size: 1em;
+  border: none;
+  height: 35px;
+  padding: 0 20px 1px 20px;
+  background-color: #FB3B49;
+  text-decoration: none;
+  margin-left: -5px;
+}
+
+.button-join {
+  color: black;
+  font-size: 1em;
+  border: 1px solid black;
+  height: 35px;
+  padding: 0 10px 0 10px;
+  margin-left: 20px;
+  background-color: white;
+  text-decoration: none;
+}
+
+.button-login {
+  color: black;
+  font-size: 1em;
+  border: 1px solid white;
+  height: 35px;
+  padding: 0 10px 0 10px;
+  margin-left: 0;
+  margin-right: -50px;
+  background-color: white;
+  text-decoration: none;
+}
+/* Section 1 */
+#discover {
+  padding: 20px 0 40px 0;
+  height: 600px; 
+  background-color: rgba(232,217,217,.3);
+  display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+#discover h1 {
+  text-align: center;
+  font-size: 5em;
+  font-weight: bold;
+  padding-top: 80px;
+}
+
+#discover article {
+  text-align: center;
+  font-size: 1.5em;
+  margin-left: 200px;
+  margin-right: 200px;
+  margin-bottom: 100px;
+  text-justify: justify;
+}
+
+#discover .central-buttons {  
+  margin-left: 150px;
+  margin-right: 150px;
+  font-size: 1.5em;
+  display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	justify-content: space-between;
+	align-items: center;
+	align-content: stretch;
+}
+
+.button1 {
+  color: white;
+  border: 1px solid #FB3B49;
+  padding: 20px 70px 20px 70px;
+  margin: 20px 20px 20px 20px;
+  background-color: #FB3B49;
+  text-decoration: none;
+  box-shadow: 8px 8px 0 rgba(251,59,73,.2);
+}
+
+.button2 {
+  color: black;
+  background-color: white;
+  border: 1px solid white;
+  margin: 20px 20px 20px 20px;
+  padding: 20px 70px 20px 70px;
+  text-decoration: none;
+}
+
+/* Section 2 */
+#slogan {
+  text-align: center;
+}
+
+#slogan h3 {
+  background-color: rgba(255,204,53,.4);
+  text-align: center;
+  font-size: 2.5em;
+  transform: skew(9deg,0deg);
+  font-style: italic;
+  margin: 90px 10px 30px 10px;
+}
+
+#slogan article {
+  text-align: center;
+  font-size: 1.2em;
+  margin-left: 360px;
+  margin-right: 360px;
+  margin-bottom: 200px;
+  text-justify: justify;
+}
+
+#slogan img {
+   margin: 90px auto 0 auto;
+   width: 350px;
+   height: 350px;
+   text-align: center;
+}
+
+#slogan {
+  display: flex;
+  flex-direction: column;
+	flex-wrap: nowrap;
+	justify-content: space-between;
+	align-items: center;
+	align-content: space-around;
+}
+
+/* Section 3 */
+.feature-box {
+  font-size: 1em;
+  text-align: left;
+  width: 15%;
+  height: 150px;
+}
+
+#features h4 {
+  color: #ED1C24;
+}
+
+.feature-box img {
+  width: 40px;
+}
+
+#features .feature-list {
+  display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	justify-content: space-evenly;
+	align-items: center;
+	align-content: center;
+}
+
+#features .button3 {
```

Comment:
This button is bigger than in the example. No major problem, just a detail that could be improved. :)

Reply:
Sure, I'll adapt it :) 
I noticed though, that I get different results according to the size of the screen. Is there a way to resize the content according to the screen resolution, for example, if I watch it on my laptop or my phone?